### DISCUSSION TITLE
[@types/ws]: Type 'Server' is not generic.

### DISCUSSION THREAD

**Discussion by JansenMoscol** [DefinitelyTyped/DefinitelyTyped#65720] [Issues with a @types package]
node_modules/@types/ws/index.d.ts(328,18): error TS2315: Type 'Server' is not generic.
node_modules/@types/ws/index.d.ts(328,34): error TS2315: Type 'Server' is not generic.

[a comment by an automated account is not shown]

**Comment by vansergen** [comment]
Related discussion - #65725
